common/sgx/eeid.c

```diff
@@ -794,21 +800,20 @@ oe_result_t oe_eeid_evidence_ntoh(
     oe_eeid_evidence_t* evidence)
 {
     oe_result_t result = OE_UNEXPECTED;
+    const uint8_t* position = buffer;
+    size_t remaining = buffer_size, data_size;
```

Comment:
Style: please split variable declarations with initializations apart from those without for readability.

```suggestion
    size_t remaining = buffer_size;
    size_t data_size = 0;
```